Misc/NEWS.d/next/Library/2020-09-29-16-23-54.bpo-41876.QicdDU.rst

```diff
@@ -0,0 +1,2 @@
+Change in :class:`tkinter.font.Font`, now representation of font object contains
+font name instead of object address.
```

Comment:
This is a very long description! Can we shorten it to something like "Tkinter font class repr uses font name"?